internal/pkg/tfconf/module.go

```diff
@@ -122,20 +126,57 @@ func CreateModule(options *Options) (*Module, error) {
 		}
 	}
 
+	options.OutputValuesPath = os.Getenv("FOO")
+	// if OutputValuesPath is empty, set the default and generate the file
+	if options.OutputValues && options.OutputValuesPath == "" {
+		err := os.Setenv("FOO", "terraform-outputs.json")
```

Comment:
I'm completely against the use of environment variable. Actually to rephrase if environment variable should be used they should be immediately parsed right from the get go (a.k.a inside `cmd/root.go`) and the parsed values should be passed down.

But in this particular case I cannot think of a reason where we should need to use this. Having a flag is extremely visible and self-explanatory but environment variables are not and error prone. Also autocomplete works for the flags and not the environment variables.

Reply:
Refactored with your original suggestions!